x-pack/plugins/security_solution/public/common/components/risk_score_over_time/index.test.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+import { RiskScoreOverTime } from '.';
+import { TestProviders } from '../../mock';
+
+describe('Host Risk Flyout', () => {
+  it('renders', () => {
+    const { queryByTestId } = render(
+      <TestProviders>
+        <RiskScoreOverTime
+          riskScore={[]}
+          loading={false}
+          from={'2020-07-07T08:20:18.966Z'}
+          to={'2020-07-08T08:20:18.966Z'}
+          queryId={'test_query_id'}
+          title={'test_query_title'}
+          toggleStatus={true}
+        />
+      </TestProviders>
+    );
+
+    expect(queryByTestId('RiskScoreOverTime')).toBeInTheDocument();
```

Comment:
instead of just checking the render, can we check that the formatter functions are being called? for example, check that `LineSeries` is receiving the proper `tickFormat` prop?

Reply:
I agree that it could have some more tests. For example, I could write a test that checks if the rendered data is formatted. 

> check that LineSeries is receiving the proper tickFormat prop

This isn't trivial with react-testing-library, because the library is designed to hide implementation details, so it doesn't provide a way to assert that a component receives a property. 

The only way I know to go around this limitation is to use jest to mock `LineSeries` and assert that it is called with the property.

Reply:
Here it is. Could you take a look?
https://github.com/elastic/kibana/pull/130256/commits/249862e6e6e79222ec3fc8762cd0ffda6d2fa43d

Reply:
yes that is how i would have done it as well, thank you!